src/components/v2/appDetails/k8Resource/nodeDetail/DeleteEphemeralButton.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+    ToastManager,
+    ToastVariantType,
+    showError,
+    Button,
+    ButtonVariantType,
+    ComponentSizeType,
+    ButtonStyleType,
+    IndexStore,
+} from '@devtron-labs/devtron-fe-common-lib'
+import { ReactComponent as Close } from '@Icons/ic-close.svg'
+import { deleteEphemeralUrl } from './nodeDetail.api'
+import { DeleteEphemeralButtonType, ResponsePayload } from './nodeDetail.type'
+
+export const DeleteEphemeralButton = ({
+    containerName,
+    isResourceBrowserView,
+    selectedNamespace,
+    selectedClusterId,
+    selectedPodName,
+    switchSelectedContainer,
+    setContainers,
+    params,
```

Comment:
Can we get params from `useParams` here. instead of getting it from props?
Also reduce the props which are available from url